docs/introduzione/README.md

```diff
@@ -0,0 +1,21 @@
+# Il Libro Open Source
+
+Chiunque tu sia, da qualunque posto tu provenga, benvenuto in un viaggio incredibile nel mondo dello sviluppo software.
+
+In queste pagine approfondiremo assieme l'essenza di uno sviluppatore, non puntando alla padronanza tecnica di un linguaggio di programmazione o di uno specifico framework, ma a una comprensione più profonda dei valori che sono alla base di questo mestiere.
```

Comment:
sviluppatore---*del mondo dello sviluppo

Reply:
Fa perdere un po' il senso, non mi vengono in mente altre forme.